src/components/input/input.styles.ts

```diff
@@ -261,7 +274,7 @@ export default css`
     display: inline-flex;
     align-items: center;
     justify-content: center;
-    font-size: inherit;
+    font-size: var(--sl-font-size-large);
```

Comment:
This change worries me a bit. Are we completely sure we always want it to be large text?